tests/py3_driver_testers.txt

```diff
@@ -343,7 +343,8 @@ Yaesu_FT-1D_R,+Yaesu_FT3D_R,30-Dec-2022
 Yaesu_FT2D_R,+Yaesu_FT3D_R,30-Dec-2022
 Yaesu_FT2D_Rv2,+Yaesu_FT3D_R,30-Dec-2022
 Yaesu_FT3D_R,@kk7ds,30-Dec-2022
-Yaesu_FT-450D,@Old-Phart,01-Jun-2023
```

Comment:
Are you testing FT-450 and FT-450D? If not, you should not change this line and only add your FT-450 report line. Also, please put it above the FT-450D so it sorts properly :)